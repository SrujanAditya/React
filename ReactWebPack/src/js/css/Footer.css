.foot {
    background-color:black;
    width : 100%;
    bottom : 0px;
    position: fixed
}