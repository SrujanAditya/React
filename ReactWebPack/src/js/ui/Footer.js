import React from 'react';
import '../css/Header.css';
import '../css/Footer.css';
 
const Footer = () => (
    <footer className="foot">
        <label className = "netflixText" >netflixroulette</label>
    </footer>
);

export default Footer;