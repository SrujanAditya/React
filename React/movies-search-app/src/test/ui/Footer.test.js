import React from 'react';
import Footer from '../../js/ui/Footer';
import renderer from 'react-test-renderer';

describe('ui', () => {
    it('Footer component renders correctly', () => {
        const FooterComp = renderer.create(<Footer/>).toJSON();
        expect(FooterComp).toMatchSnapshot();
    }) 
})
