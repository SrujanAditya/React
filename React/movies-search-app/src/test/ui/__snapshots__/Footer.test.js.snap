// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ui Footer component renders correctly 1`] = `
<footer
  className="foot"
>
  <label
    className="netflixText"
  >
    netflixroulette
  </label>
</footer>
`;
